client/gameObjects/Skeleton.js

```diff
@@ -146,7 +160,18 @@ export default class Skeleton extends Phaser.GameObjects.Image {
       }
     }
 
-    if (this.isMainPlayer && destTile !== null) {
+    if (this.isMainPlayer) {
+      Skeleton.selectedPlayer = selectedPlayer;
+      Skeleton.fightTile = fight;
+    }
+
+    if (Skeleton.fightTile) {
+      this.tileFight.toggleVisible(this.name === Skeleton.selectedPlayer);
+    } else {
+      this.tileSelected.toggleVisible(this.name === Skeleton.selectedPlayer);
+    }
```

Comment:
This is the only place where `Skeleton.fightTile` is used, we can get rid of it and use `fight` directly.